### ISSUE TITLE
Process finished with exit code -1073741819 (0xC0000005) 

### ISSUE THREAD

**Issue by DenisaLicu** [PaddlePaddle/PaddleOCR#2419]
I am trying to fine tune the model using the pre-trained model and a custom character dictionary.  When loading the dataset I get this error, that I couldn't figure it out: Process finished with exit code -1073741819 (0xC0000005). I only have 6 images in the training data so a memory overload wouldn't be possible.

This is the complete output:

Architecture :
[2021/04/08 16:41:30] root INFO:     Backbone :
[2021/04/08 16:41:30] root INFO:         model_name : large
[2021/04/08 16:41:30] root INFO:         name : MobileNetV3
[2021/04/08 16:41:30] root INFO:         scale : 0.5
[2021/04/08 16:41:30] root INFO:     Head :
[2021/04/08 16:41:30] root INFO:         fc_decay : 0
[2021/04/08 16:41:30] root INFO:         name : CTCHead
[2021/04/08 16:41:30] root INFO:     Neck :
[2021/04/08 16:41:30] root INFO:         encoder_type : rnn
[2021/04/08 16:41:30] root INFO:         hidden_size : 96
[2021/04/08 16:41:30] root INFO:         name : SequenceEncoder
[2021/04/08 16:41:30] root INFO:     Transform : None
[2021/04/08 16:41:30] root INFO:     algorithm : CRNN
[2021/04/08 16:41:30] root INFO:     model_type : rec
[2021/04/08 16:41:30] root INFO: Eval :
[2021/04/08 16:41:30] root INFO:     dataset :
[2021/04/08 16:41:30] root INFO:         data_dir : ../train_data/ic15_data/
[2021/04/08 16:41:30] root INFO:         label_file_list : ['../train_data/ic15_data/rec_gt_test.txt']
[2021/04/08 16:41:30] root INFO:         name : SimpleDataSet
[2021/04/08 16:41:30] root INFO:         transforms :
[2021/04/08 16:41:30] root INFO:             DecodeImage :
[2021/04/08 16:41:30] root INFO:                 channel_first : False
[2021/04/08 16:41:30] root INFO:                 img_mode : BGR
[2021/04/08 16:41:30] root INFO:             CTCLabelEncode : None
[2021/04/08 16:41:30] root INFO:             RecResizeImg :
[2021/04/08 16:41:30] root INFO:                 image_shape : [3, 32, 100]
[2021/04/08 16:41:30] root INFO:             KeepKeys :
[2021/04/08 16:41:30] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/04/08 16:41:30] root INFO:     loader :
[2021/04/08 16:41:30] root INFO:         batch_size_per_card : 1
[2021/04/08 16:41:30] root INFO:         drop_last : False
[2021/04/08 16:41:30] root INFO:         num_workers : 4
[2021/04/08 16:41:30] root INFO:         shuffle : False
[2021/04/08 16:41:30] root INFO:         use_shared_memory : False
[2021/04/08 16:41:30] root INFO: Global :
[2021/04/08 16:41:30] root INFO:     cal_metric_during_train : True
[2021/04/08 16:41:30] root INFO:     character_dict_path : ../ppocr/utils/dict/innovalor_dict.txt
[2021/04/08 16:41:30] root INFO:     character_type : ch
[2021/04/08 16:41:30] root INFO:     checkpoints : None
[2021/04/08 16:41:30] root INFO:     debug : False
[2021/04/08 16:41:30] root INFO:     distributed : False
[2021/04/08 16:41:30] root INFO:     epoch_num : 10
[2021/04/08 16:41:30] root INFO:     eval_batch_step : [0, 10]
[2021/04/08 16:41:30] root INFO:     infer_img : ../doc/imgs_words_en/word_10.png
[2021/04/08 16:41:30] root INFO:     infer_mode : False
[2021/04/08 16:41:30] root INFO:     log_smooth_window : 2
[2021/04/08 16:41:30] root INFO:     max_text_length : 25
[2021/04/08 16:41:30] root INFO:     pretrained_model : ../output/rec/ic15/best_accuracy
[2021/04/08 16:41:30] root INFO:     print_batch_step : 1
[2021/04/08 16:41:30] root INFO:     save_epoch_step : 3
[2021/04/08 16:41:30] root INFO:     save_inference_dir : None
[2021/04/08 16:41:30] root INFO:     save_model_dir : ../output/rec/ic15/
[2021/04/08 16:41:30] root INFO:     use_gpu : True
[2021/04/08 16:41:30] root INFO:     use_space_char : True
[2021/04/08 16:41:30] root INFO:     use_visualdl : False
[2021/04/08 16:41:30] root INFO: Loss :
[2021/04/08 16:41:30] root INFO:     name : CTCLoss
[2021/04/08 16:41:30] root INFO: Metric :
[2021/04/08 16:41:30] root INFO:     main_indicator : acc
[2021/04/08 16:41:30] root INFO:     name : RecMetric
[2021/04/08 16:41:30] root INFO: Optimizer :
[2021/04/08 16:41:30] root INFO:     beta1 : 0.9
[2021/04/08 16:41:30] root INFO:     beta2 : 0.999
[2021/04/08 16:41:30] root INFO:     lr :
[2021/04/08 16:41:30] root INFO:         learning_rate : 0.0005
[2021/04/08 16:41:30] root INFO:     name : Adam
[2021/04/08 16:41:30] root INFO:     regularizer :
[2021/04/08 16:41:30] root INFO:         factor : 0
[2021/04/08 16:41:30] root INFO:         name : L2
[2021/04/08 16:41:30] root INFO: PostProcess :
[2021/04/08 16:41:30] root INFO:     name : CTCLabelDecode
[2021/04/08 16:41:30] root INFO: Train :
[2021/04/08 16:41:30] root INFO:     dataset :
[2021/04/08 16:41:30] root INFO:         data_dir : ../train_data/ic15_data/
[2021/04/08 16:41:30] root INFO:         label_file_list : ['../train_data/ic15_data/rec_gt_train.txt']
[2021/04/08 16:41:30] root INFO:         name : SimpleDataSet
[2021/04/08 16:41:30] root INFO:         transforms :
[2021/04/08 16:41:30] root INFO:             DecodeImage :
[2021/04/08 16:41:30] root INFO:                 channel_first : False
[2021/04/08 16:41:30] root INFO:                 img_mode : BGR
[2021/04/08 16:41:30] root INFO:             CTCLabelEncode : None
[2021/04/08 16:41:30] root INFO:             RecResizeImg :
[2021/04/08 16:41:30] root INFO:                 image_shape : [3, 32, 100]
[2021/04/08 16:41:30] root INFO:             KeepKeys :
[2021/04/08 16:41:30] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/04/08 16:41:30] root INFO:     loader :
[2021/04/08 16:41:30] root INFO:         batch_size_per_card : 1
[2021/04/08 16:41:30] root INFO:         drop_last : True
[2021/04/08 16:41:30] root INFO:         num_workers : 8
[2021/04/08 16:41:30] root INFO:         shuffle : True
[2021/04/08 16:41:30] root INFO:         use_shared_memory : False
[2021/04/08 16:41:30] root INFO: train with paddle 2.0.0 and device CUDAPlace(0)
[2021/04/08 16:41:30] root INFO: Initialize indexs of datasets:['../train_data/ic15_data/rec_gt_train.txt']
[2021/04/08 16:41:30] root INFO: Initialize indexs of datasets:['../train_data/ic15_data/rec_gt_test.txt']
W0408 16:41:30.284492 13440 device_context.cc:362] Please NOTE: device: 0, GPU Compute Capability: 7.5, Driver API Version: 11.1, Runtime API Version: 10.2
W0408 16:41:30.287503 13440 device_context.cc:372] device: 0, cuDNN Version: 7.6.
[2021/04/08 16:41:32] root INFO: load pretrained model from ['../output/rec/ic15/best_accuracy']
[2021/04/08 16:41:32] root INFO: train dataloader has 3 iters, valid dataloader has 3 iters
[2021/04/08 16:41:32] root INFO: During the training process, after the 0th iteration, an evaluation is run every 10 iterations
[2021/04/08 16:41:32] root INFO: Initialize indexs of datasets:['../train_data/ic15_data/rec_gt_train.txt']
[2021/04/08 16:41:32] root INFO: epoch: [1/10], iter: 1, lr: 0.000500, loss: 200.846405, acc: 0.000000, norm_edit_dis: 0.000000, reader_cost: 0.01001 s, batch_cost: 0.20338 s, samples: 2, ips: 9.83362
[2021/04/08 16:41:32] root INFO: epoch: [1/10], iter: 2, lr: 0.000500, loss: 199.678284, acc: 0.000000, norm_edit_dis: 0.000000, reader_cost: 0.00000 s, batch_cost: 0.05660 s, samples: 1, ips: 17.66886

Process finished with exit code -1073741819 (0xC0000005)

Can someone help me, please?

**Comment by MissPenguin** [comment]
6 images are too few

**Comment by DenisaLicu** [comment]
It was a Windows problem, running on Linux works
